feature/dashboard/src/main/java/com/simprints/feature/dashboard/views/SyncCardView.kt

```diff
@@ -27,6 +27,7 @@ internal class SyncCardView : MaterialCardView {
     var onSelectNoModulesButtonClick: () -> Unit = {}
     var onOfflineButtonClick: () -> Unit = {}
     var onLoginButtonClick: () -> Unit = {}
+    var onSettingsButtonClick: () -> Unit = {}
```

Comment:
This will add a couple of hundred lines more to Oleksii's PR :(

Reply:
Or to mine :( But hopefully not in the hundreds!